### PR TITLE
Update the Full-Text Search webservice documentation

### PR DESCRIPTION
**Pull request by sbrunner**
Linked to #1840

### CODE REVIEW

File: doc/developer/webservices.rst [hunk 1]
```diff
@@ -297,7 +297,10 @@ Parameters
 ----------
 
 * ``query``: Text to search.
-* ``callback``: Name of the callback function.
+* ``limit``: The maximum number of results (Optional).
```
Comment (ger-benjamin) [inline doc/developer/webservices.rst]: **o**ptional (no capital 'o')
Same lower.

File: doc/developer/webservices.rst [hunk 2]
```diff
@@ -288,7 +288,7 @@ Success JSON result
    }
 
 
-Full Text Search
+Full-Text Search
 ================
```
Comment (ger-benjamin) [inline doc/developer/webservices.rst]: +1 '=' ?
(Perhaps the line of `====` is to short)

File: doc/developer/webservices.rst [hunk 3]
```diff
@@ -307,7 +310,15 @@ A GeoJSON of a feature collection with the properties:
...
 * ``layer_name``: Layer to display.
 * ``params``: :ref:`integrator_fulltext_search_params` to set.
+* ``actions``: List of actions.
 
+Action
+~~~~~~
+
+A dictionary with:
+
+* ``action``: the action: (add_theme|add_group|add_layer).
```
Comment (ger-benjamin) [inline doc/developer/webservices.rst]: **T**he action (capital 'T'). Same lower.

File: doc/developer/webservices.rst (hunk 2 above)
Comment (sbrunner) [inline doc/developer/webservices.rst]: no ...

**Comment by ochriste** [comment]
lgtm

**Pull request merged**